secure_redis/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
what is this file for? Its a very odd way to use a migration. I think we should (if necessary) include this as a management command instead.